src/Semantic/CLI.hs

```diff
@@ -187,6 +188,12 @@ shaReader = eitherReader parseSha
 filePathReader :: ReadM File
 filePathReader = fileForPath <$> str
 
+path :: (Path.PartClass.FileDir fd) => ReadM (Path.AbsRel fd)
+path = eitherReader Path.parse
+
+pathOption :: Path.PartClass.FileDir fd => Mod OptionFields (Path.AbsRel fd) -> Parser (Path.AbsRel fd)
+pathOption = option path
```

Comment:
These were copied directly from the `pathtype` documentation (I guess they didn’t want `pathtype` to depend on `optparse-applicative`.) They fail immediately if a path cannot be converted.

Reply:
This is really cool 👍